base/shell/explorer/trayclock.cpp

```diff
@@ -38,6 +38,9 @@ const struct
 const UINT ClockWndFormatsCount = _ARRAYSIZE(ClockWndFormats);
 
 #define CLOCKWND_FORMAT_COUNT ClockWndFormatsCount
+#define CLOCKWND_FORMAT_TIME 0
+#define CLOCKWND_FORMAT_DAY  1
```

Comment:
This define is unused: it could be commented out for the time being, or a comment added.

Reply:
Commented out ```CLOCKWND_FORMAT_DAY``` and left a comment to uncomment it if needed.